New package: AstroUniverse v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: AstroUniverse
- Repository: https://github.com/GenAstro/Epicycle
- Created by: @stevenhughes73
- Version: v0.1.0
- Commit: 751cd5f385aab718d965f20e87a30fe20fcdbc84
- Reviewed by: @stevenhughes73
- Reference: https://github.com/GenAstro/Epicycle/commit/751cd5f385aab718d965f20e87a30fe20fcdbc84#commitcomment-171321150
- Description: An application for space systems with a nod to the giants who came before us and to how far we’ve come. 
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
AstroUniverse is part of the Epicycle space mission design and navigation system.
Initial release - celestial body models and solar system data.
README - https://github.com/GenAstro/Epicycle/blob/main/AstroUniverse/README.md
docs - https://genastro.github.io/Epicycle/AstroUniverse/dev/

This package is part of a larger monorepo. The original PR removed the deps to avoid sequential registration and three day timers. This registration contains all deps, which only depends on EpicycleBase which is already merged (see here https://github.com/JuliaRegistries/General/pull/142542#event-21019565512)
`````
<!-- END RELEASE NOTES -->

<!-- 80cc8f6b60f354a8c40306cc5f8995f3e1bd9a763847aad2fd25ea3e49d0f9b4cade407b275103341b5e463d283cef31a3639a04b4e3afc856368148c5eac4c76acb0d9bb254fbfd79dfd8ead400e568717df3f7b49c9fc1a27109141270cd9ef5043fb0dc953c929e23196c1a30f06cb366b495ce3e859b0b148e70983825e06fad52f86e45fab07bba20717f2a65f76a90350d8b8251c7e0e5f3a34a9f4ffc2a93029d9b29a0f3049a1fa0e69e966f94974253924c2ea90123d1af16b1f37234f69d1c99ee42a43b6320312423b73061b43fcb2fdebfdf100f279238d62d81 -->

aplavin commented:
Personally, I still stand behind my concern with naming, most notably for the AstroBase package. This AstroUniverse one is more borderline, but you seem to want the same prefix for all.

stevenhughes73 commented:
> Personally, I still stand behind my concern with naming, most notably for the AstroBase package. This AstroUniverse one is more borderline, but you seem to want the same prefix for all.

@aplavin   I renamed AstroBase to EpicycleBase a few days ago.  Can you please unblock this PR now.  I would appreciate it. 

[noblock]

JuliaRegistrator committed:
New package: AstroUniverse v0.1.0

UUID: 4bd44dca-1e45-43d2-b940-3f88da59ea6f
Repo: https://github.com/GenAstro/Epicycle.git
Tree: fc7bf3fde77274077ebb6bf5b8aab774ceb188e3

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
